rules/order.js

```diff
@@ -59,18 +107,59 @@ module.exports = {
       order: 'strict',
       properties: [
         'background',
+        'background-attachment',
+        'background-clip',
         'background-color',
-        'background-size',
-        'background-position',
+        'background-image',
         'background-repeat',
+        'background-position',
+        'background-size',
```

Comment:
The background stuff is all alphabetical except that -repeat and -position are flipped. Is it meant to be alphabetical?

If not, I notice you swapped the order of -position and -size. What's the rationale there? (or was it unintentional?)